books/free-programming-books-subjects.md

```diff
@@ -227,6 +227,7 @@ Books that cover a specific programming language can be found in the [BY PROGRAM
 * [Computer Science I](https://cse.unl.edu/~cbourke/ComputerScienceOne.pdf) - Chris Bourke (PDF)
 * [Computer Science II](https://cse.unl.edu/~cbourke/ComputerScienceTwo.pdf) - Chris Bourke (PDF)
 * [CS Principles: Big Ideas in Programming](https://www.openbookproject.net/books/StudentCSP/) - Mark Guzdial, Barbara Ericson (HTML)
+* [Pull Requests and Code Review](https://scs.tl/book-pr) – Sebastien Castiel (Markdown, ePub, PDF, Kindle)
```

Comment:
the kindle version is not free. Since there are multiple formats, best to just omit the format notes.
```suggestion
* [Pull Requests and Code Review](https://scs.tl/book-pr) – Sebastien Castiel
```